eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from ..lib import manageevent as me
+from ..lib import readECF as rd
+import glob, os
+from ..lib import sort_nicely as sn
+import matplotlib.pyplot as plt
+from importlib import reload
+
+def fitJWST(eventlabel, workdir, speclc_dir, fit_par, run_par_file, meta):
+    # load savefile
+    if meta == None:
+        meta = me.load(speclc_dir + '/S4_' + eventlabel + '_Meta_Save.dat')
+
+    #read in run params
+    import eureka.S5_lightcurve_fitting.parameters as p
+    reload(p)
+    run_par=p.Parameters(param_file=run_par_file)
+    # Create directories for Stage 4 processing
+    files = glob.glob(os.path.join(speclc_dir + "/speclc", "*.txt"))
+    files = sn.sort_nicely(files)
+    if run_par.run_verbose.value:
+        print(files)
+    t0_offset = run_par.toffset.value#59694
+    for f in [files[0]]:
+        t_bjdtdb, flux, flux_err = np.loadtxt(f, skiprows=1).T
+        t_bjdtdb = t_bjdtdb - t0_offset
+        flux = flux / np.median(flux[:200])
+        flux_err = flux_err/800000000/3
+
+        import eureka.S5_lightcurve_fitting.lightcurve as lc
+        reload(lc)
+        wasp43b_lc = lc.LightCurve(t_bjdtdb, flux, unc=flux_err, name='WASP-43b')
+
+        # Get the orbital parameters
+        from eureka.S5_lightcurve_fitting.utils import get_target_data
+        wasp43b_params, url = get_target_data('WASP-43b')
+
+        # Set the intial parameters
+        params = p.Parameters(param_file=fit_par)
+        # params.rp = wasp43b_params['Rp/Rs'] + 0.02, 'free', 0.1, 0.2
```

Comment:
If these commented out lines no longer serve any purpose, it'd be best to cut them out to keep files smaller and easier to read through